crates/bdk/src/wallet/coin_selection.rs

```diff
@@ -718,6 +677,57 @@ impl BranchAndBoundCoinSelection {
     }
 }
 
+pub(crate) fn single_random_draw(
+    required_utxos: Vec<WeightedUtxo>,
+    optional_utxos: Vec<WeightedUtxo>,
+    target_amount: u64,
+    drain_script: &Script,
+    fee_rate: FeeRate,
+    rng: &mut impl RngCore,
+) -> CoinSelectionResult {
```

Comment:
This is a public function without docstrings.
I know that the original one hasn't, but it was a non-public internal function.

Can we put some docstrings here?

Reply:
`pub(crate)` is not visible to users. I can add developer comments?

Reply:
Ok, then sorry about the nagging. It's fine as it is.

Reply:
> Can we put some docstrings here?

I see what you're saying, I think it could go either way